include/chimera/util.h

```diff
@@ -311,6 +311,14 @@ std::pair<unsigned, unsigned> getFunctionArgumentRange(
  */
 bool hasNonPublicParam(const clang::CXXMethodDecl *decl);
 
+/**
+ * Converts clang operator type to the operator name in Python.
+ *
+ * Return true on success.
+ */
+bool getPythonOperatorSpelling(clang::OverloadedOperatorKind kind,
+                               std::string &str);
```

Comment:
I propose the following type that always returns some string.  Since we already have a check for which kinds of operators we support, there is no point in doing the check twice.

We could possibly raise a runtime error in the `default` case since this would be an internal logic error if the two did not match anyway.

```suggestion
std::string getPythonOperatorName(clang::OverloadedOperatorKind kind);
```

Reply:
> We could possibly raise a runtime error in the `default` case since this would be an internal logic error if the two did not match anyway.

This implies that we raise runtime error for unsupported or implemented operator types. Is this the intention? Not sure if I get what you mean by "the two did not match anyway".

Reply:
We already have this `if` block which is checking which operators we pass to this function anyway:
https://github.com/personalrobotics/chimera/pull/322/files#diff-3f8c8328f0a509fbeb0a6d422bf8a85bR443

So I don't see the point in _also_ checking the same thing inside the `getPythonOperatorName()` function, since we can just make sure we implement anything that makes it past this check anyway.  Then if we hit the default case, we must have some sort of bug where things are getting past the check that should not be.

Reply:
Sounds fair to me!